src/Client/App.fs

```diff
@@ -14,6 +14,8 @@ open Client.Messages
 open Elmish.Browser.UrlParser
 open Elmish.HMR
 
+JsInterop.importAll "whatwg-fetch"
```

Comment:
There's also `JsInterop.importSideEffects "whatwg-fetch"` for importing something without assigning it into a variable. In this case it has the same effect, just maybe the semantics is a bit clearer.